src/my_project/database/gui_repo_worker.py

```
@@ -23,19 +17,16 @@ class RepoWorker:
 
-    COVER_FOLDER = ROOT_DIR / ".covers"
-
-    def __init__(self):
-        """
-        Intiates the class instance.
-        """
+    def __init__(self, config_manager: ConfigManager):
+        """Intiates the class instance."""
+        self.config_manager = config_manager
 
     def __enter__(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__enter__`

(ANN204)